source/code/plugins/out_oms_blob.rb

```diff
@@ -136,16 +125,42 @@ def get_blob_uri(container_type, data_type, custom_data_type, suffix)
 
       # remove the BOM (Byte Order Marker)
       clean_body = body.encode(Encoding::UTF_8, :invalid => :replace, :undef => :replace, :replace => "")
-      blob_uri_json = JSON.parse(clean_body)
-      return URI.parse( blob_uri_json["Uri"] )
-    end # get_blob_uri
+      return JSON.parse(clean_body)
+    end # request_blob_json
+
+    # get the blob SAS URI and committed blocks from ODS
+    # parameters
+    #   container_type: string. ContainerType of the data
+    #   data_type: string. DataTypeId of the data
+    #   custom_data_type: string. CustomDataType of the CustomLog
+    #   suffix: string. Suffix of the blob
+    # returns:
+    #   URI. blob SAS URI
+    #   string[]. a list of committed blocks
+    def get_blob_uri_and_committed_blocks(container_type, data_type, custom_data_type, suffix)
```

Comment:
Is there any reason we'd need to use it though? I can read it from the JSON, but I don't think any of our code relies on that property.